sdk/keyvault/azure-keyvault-certificates/pom.xml

```diff
@@ -11,8 +11,8 @@
   </parent>
 
   <groupId>com.azure</groupId>
```

Comment:
Lets be sure to update the directory name to match the name change as well.

Reply:
yup, that's my next commit.